projects/optic/src/__tests__/integration/ruleset-publish.test.ts

```diff
@@ -1,23 +1,36 @@
-import { runOptic, setupWorkspace, normalizeWorkspace } from './integration';
-
+import {
+  runOptic,
+  setupWorkspace,
+  normalizeWorkspace,
+  setupTestServer,
+} from './integration';
 jest.setTimeout(30000);
-// TODO replace this with real mocks when connected to backend
-jest.mock('node-fetch');
+
+setupTestServer(({ url, method }) => {
```

Comment:
Really like this model for testing against a mock server. Pretty smart to just allow it to take in a function that gets the url and method like that.